indexer/src/main/java/bio/terra/tanagra/indexing/job/bigquery/WriteEntityLevelDisplayHints.java

```diff
@@ -173,11 +173,10 @@ public void run(boolean isDryRun) {
     // Do a single insert to BQ for all the hint rows.
     SqlParams sqlParams = new SqlParams();
     List<String> insertRowSqls = new ArrayList<>();
-    for (int i = 0; i < insertRows.size(); i++) {
-      List<Literal> insertRow = insertRows.get(i);
+    for (List<Literal> insertRow : insertRows) {
       List<String> paramNames = new ArrayList<>();
-      for (int j = 0; j < insertRow.size(); j++) {
-        paramNames.add("@" + sqlParams.addParam("valr" + i + "c" + j, insertRow.get(j)));
+      for (Literal literal : insertRow) {
+        paramNames.add("@" + sqlParams.addParam("valr", literal));
```

Comment:
No need to pass in ints since the param names are uniquely generated internally.